### ISSUE TITLE
role-has-required-aria-props should bail on components

### ISSUE THREAD

**Issue by jessebeach** [evcohen/eslint-plugin-jsx-a11y#303]
Currently, `role-has-required-aria-props` is flagging components, not just HTML elements. It should only flag HTML elements.

Wrong

```
<MyComponent role="combobox" />
// !! Components is missing aria-controls, aria-expanded
```

Correct

```
<div role="combobox" />
// !! Components is missing aria-controls, aria-expanded
```
